src/gen/validation_error.rs

```diff
@@ -73,6 +77,11 @@ impl From<ErrorCode> for u32 {
             ErrorCode::InvalidCoinAmount => 10,
             ErrorCode::AssertHeightAbsolute => 14,
             ErrorCode::AssertHeightRelative => 13,
+            // TODO: introduce specific error codes on the python side
+            ErrorCode::AssertBeforeSecondsAbsolute => 117,
+            ErrorCode::AssertBeforeSecondsRelative => 117,
+            ErrorCode::AssertBeforeHeightAbsolute => 117,
+            ErrorCode::AssertBeforeHeightRelative => 117,
```

Comment:
I think it's okay to just declare them all as new values here and then we can force python to be compliant later.

Reply:
ok, I'll do that. I was worried about forgetting to add those error codes to python later, but I've already started a "bump-chia_rs" branch, locally, where I'm collecting these kinds of things.